CHANGELOG.md

```diff
@@ -3,13 +3,20 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
-## v0.4
-
+## [0.4] - 2016-04-28
 ### Added
 - Added grouped bar charts
+- Dashboard examples
```

Comment:
I would put the following under a support section:

### Support
- Dashboard examples
- Map chart example
- Deploy runs tests in Travis
- Update tooling

and if this is a docs fix, then also:
- Fix leaflet map location

In other words, make a clear distinction between what changed in the source and what changed in the docs/tooling.